[IMP] accounting/l10n_uy: clarify onboarding (#11715)
odoo/documentation: content/applications/finance/fiscal_localizations/uruguay.rst

proceeding steps.

Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
Configuration --> Settings`. Scroll down to the :guilabel:`Uruguayan Localization` section, select

> the environment (Production or Testing) and click on :guilabel:`Create Uruware Account`.

Reviewer: We can use :guilabel: here for these radio buttons :)

@fvz-odoo What about the "Demo" option? How is this option used?

```suggestion
the environment (:guilabel:`Production` or :guilabel:`Testing`) and click on :guilabel:`Create
Uruware Account`.
```

Author: > We can use :guilabel: here for these radio buttons :)
> 
> @fvz-odoo What about the "Demo" option? How is this option used?

Demo mode does not work with Uruware, it is only for internal validations. We mention a bit after what it's for in line 155. Do you think it is worth to mention that demo mode does not need an Uruware account?

Reviewer: I see! Yes perhaps it would be a good idea to add onto the line 155 description: "Using :guilabel:`Demo mode` does not require a Uruware account."

Author: Hey @samueljlieber 

Perfect, I did it here d9c17bc500b6cd04c56c1353e84a9c8d1a97479b

Let me know what you think. Cheers!

Upon doing so, an email is sent within 48hrs to the address associated with your Odoo subscription 
with the password to enter Uruware's portal and set up your account.

.. tip::
   Your :guilabel:`Tax ID` needs to be setup to be able to create your account. The email with the
   credentials is not immediate, it might take up to 48hrs for the account to be created. The 
   username will be the email address or RUT.odoo (eg. 213344556677). The password sent will expire 

>    in 24hrs, if this is the case, just reset it by using the Forgot Password link in Uruware's portal.

Reviewer: ```suggestion
   in 24hrs, if this is the case, just reset it by using the *Forgot Password* link in Uruware's
   portal.
```

.. note::
   This action will create an account with Uruware with the following information:

   - Legal name (razón social)
   - RUT from the company

>    - User name which will be Odoo's subscription email or RUT.odoo, for example 213344556677.odoo

Reviewer: To avoid writing in future tense :)
```suggestion
   - User name (the Odoo subscription email or RUT.odoo. For example: 213344556677.odoo)
```

   - Odoo database link

   To ensure your account is created correctly, please add any missing information from above.

> Once the account is created and you have the credentials sent by email, be sure to configure your
> accounts directly in the Uruware `testing portal <https://odootest.ucfe.com.uy/Gestion/>`_ or
> `production portal <https://prod6109.ucfe.com.uy/Gestion/>`_:

Reviewer: Slightly rewording the sentence for clarity
```suggestion
Once the account is created, and you have received the email containing the credentials, be sure to
configure your accounts directly in the Uruware `testing portal
<https://odootest.ucfe.com.uy/Gestion/>`_ or `production portal
<https://prod6109.ucfe.com.uy/Gestion/>`_:
```

.. tip::
  - The company's :guilabel:`Tax ID` needs to be set up to be able to create an Uruware account. The email with the credentials is not immediate; it might take up to 48 hours for the account to be created.
  - The password sent expires after 24 hours. In this case, reset it by using the *Forgot Password* link in Uruware's portal.

.. note::
   This action will create an account with Uruware with the following information:

   - Legal name (razón social)
   - RUT from the company
   - Username (the Odoo subscription email or `RUT`.odoo. For example: `213344556677.odoo`)
   - Odoo database link

   To ensure your account is created correctly, please add any missing information from above.

Once the account is created and you have received the email containing the credentials, configure
your accounts directly in the Uruware `testing portal <https://odootest.ucfe.com.uy/Gestion/>`_ or 
`production portal <https://prod6109.ucfe.com.uy/Gestion/>`_:

Use the account credentials in the email to log in to the to the corresponding (`test
<https://odootest.ucfe.com.uy/Gestion/>`_ or `production <https://prod6109.ucfe.com.uy/Gestion/>`_)
portal.

> - Complete and correct the company's information
> - Add your digital certificate
> - Add your :abbr:`CAEs (Constancia de Autorización para Emisión)` for each document-type you plan to 
> issue.
> - Configure the format of the PDF to be printed and sent to your customers

Author: ```suggestion
In Uruware's portal, the following steps are needed to be able to issue invoices from Odoo:
1. Complete and correct the company's information.
2. Add your digital certificate.
3. Add your :abbr:`CAEs (Constancia de Autorización para Emisión)` for each document-type you plan to 
issue.
4. Configure the format of the PDF to be printed and sent to your customers.
```
@samueljlieber this is my suggestion, but feel free to correct it if you think it is necessary. Thanks!

Reviewer: @fvz-odoo looks good! Only changes are to add a blank line before the numbered list and to use dynamic numbering syntax :)
```suggestion
In Uruware's portal, the following steps are needed to be able to issue invoices from Odoo:

#. Complete and correct the company's information.
#. Add your digital certificate.
#. Add your :abbr:`CAEs (Constancia de Autorización para Emisión)` for each document-type you plan to 
issue.
#. Configure the format of the PDF to be printed and sent to your customers.
```

.. note::
   This data can be obtained from the Uruware portal, after configuring the :ref:`Uruware account

>    <l10n_uy/uruware-account>`. To get the :guilabel:`Uruware WS Password` go to Configuration - 
>    Company - Edit and look for the Validators and Additional Information tab and find WS Password.
>    To get the :guilabel:`Commerce Code` go to Configuration - Branches and to get the 
>    :guilabel:`Terminal Code` go to Configuration - Issuing Points.

Reviewer: Add menuselection and update formatting :)
```suggestion
   <l10n_uy/uruware-account>`.

   To get the :guilabel:`Uruware WS Password` go to :menuselection:`Configuration --> Company -->
   Edit` and look for the :guilabel:`Validators and Additional Information` tab to find
   :guilabel:`WS Password`.

   To get the :guilabel:`Commerce Code` go to :menuselection:`Configuration --> Branches`.

   To get the :guilabel:`Terminal Code` go to :menuselection:`Configuration --> Issuing Points`.
```

.. note::
   This data can be obtained from the Uruware portal, after configuring the :ref:`Uruware account
   <l10n_uy/uruware-account>`.

>    To get the :guilabel:`Uruware WS Password` go to :menuselection:`Configuration --> Company -->

Reviewer 2: ```suggestion
   To get the :guilabel:`Uruware WS Password`, go to :menuselection:`Configuration --> Company -->
```

   Edit` and look for the :guilabel:`Validators and Additional Information` tab to find
   :guilabel:`WS Password`.

   To get the :guilabel:`Commerce Code` go to :menuselection:`Configuration --> Branches`.

>    To get the :guilabel:`Terminal Code` go to :menuselection:`Configuration --> Issuing Points`.

Reviewer 2: ```suggestion
   To get the :guilabel:`Terminal Code`, go to :menuselection:`Configuration --> Issuing Points`.
```
